teachertool/src/state/reducer.ts

```diff
@@ -40,6 +40,22 @@ export default function reducer(state: AppState, action: Action): AppState {
                 },
             };
         }
+        case "SET_EVAL_RESULTS_BATCH": {
+            return {
+                ...state,
+                evalResults: action.criteriaResults,
+            };
+        }
+        case "CLEAR_ALL_EVAL_RESULT_NOTES": {
+            const evalResults = { ...state.evalResults };
+            for (const result of Object.keys(evalResults)) {
+                delete evalResults[result].notes
```

Comment:
Do we really want to delete here, or would setting it to undefined be sufficient?

Reply:
Another alternative... would setting `notes` to an empty string be preferred?